policy/modules/system/sysnetwork.if

```diff
@@ -513,6 +512,7 @@ interface(`sysnet_create_config',`
 
 	files_search_etc($1)
 	allow $1 net_conf_t:file create_file_perms;
+	allow $1 net_conf_t:lnk_file create_lnk_file_perms;
```

Comment:
As `create_file_perms` is not sufficient for creating files, I'd rather see it replaced with `create_files_pattern`. I suppose it just happens to work because of other rules bringing the required permissions. The same for lnk_files.

Reply:
Good point! I'll fix it up in a while...

Reply:
Actually, I would probably just replace `files_search_etc($1)` with `files_rw_etc_dirs($1)`. `create_files_pattern()` would just mean allowing to have `net_conf_t` directories in the chain, which isn't currently allowed in this interface (not even conditionally). All this should be refactored one day, but I think I went deep enough into the rabbit hole for now :)

Reply:
What I had in mind was `create_files_pattern($1, etc_t, net_conf_t)`. It is not clear at first glance `create_file_perms` are not sufficient so it does not give a good example to follow. Surely it works.

Reply:
Yes, but `etc_t` is defined in another module, so it shouldn't be referenced there directly. And to avoid that one would have to create some partial pattern in files.if and maybe convert the other places to use it... as I said, it would be a deep rabbit hole :)

Reply:
OK, agreed.